src/pages/iou/request/step/IOURequestStepScan/index.tsx

```diff
@@ -486,7 +486,10 @@ function IOURequestStepScan({
 
     const desktopUploadView = () => (
         <>
-            <View onLayout={({nativeEvent}) => setReceiptImageTopPosition(PixelRatio.roundToNearestPixel(nativeEvent.layout.y))}>
+            <View
+                // @ts-expect-error `LayoutChangeEvent` doesn't contain a `top` property, but it is needed in the code to set the position of the receipt image
```

Comment:
> `LayoutChangeEvent` doesn't contain a `top` property

This is not true. It has a top property but not defined in the types(I haven't closely looked at this)

https://expensify.slack.com/archives/C01GTK53T8Q/p1713526934258789?thread_ts=1712578137.578349&cid=C01GTK53T8Q